ros2component/ros2component/api/__init__.py

```diff
@@ -82,7 +81,10 @@ def get_container_components_info(*, node, remote_container_node_name):
         '{}/_container/list_nodes'.format(remote_container_node_name)
     )
     try:
-        list_nodes_client.wait_for_service()
+        if not list_nodes_client.wait_for_service(timeout_sec=5.0):
+            raise RuntimeError(
+                f'No list_nodes service found for {remote_container_node_name} container'
```

Comment:
Please add single quotes around the placeholder (and therefore use double quotes for the string literal).

Same twice below.

Reply:
Done in https://github.com/ros2/ros2cli/pull/421/commits/7f5057a27aa0abf03a011c1f08da87f94fdd70c0.